src/main.js

```diff
@@ -1,26 +1,19 @@
-const express = require('express');
```

Comment:
Have the references to  `server.js` been updated to `main.js`? For example, I still see `test/helper.js` have: `const load = require('../src/server');`.